.readthedocs.yml

```diff
@@ -0,0 +1,20 @@
+---
+version: 2
+
+build:
+  image: testing
+  apt_packages:
+    - python3-dev
+    - libmysqlclient-dev
```

Comment:
Why do we need this package in the configuration of `readthedocs`?

Reply:
There were some issues with `mysqlclient`. The readthedocs build would fail and the error message was that the version number does not exist, which is not true. I looked it up and it seems like the build environment did not include the mysql development libraries which is `libmysqlclient-dev`. Regarding `python3-dev` it might not be needed, but it was included in the instructions.

Reply:
Ok. Same issue because it needs to import the modules to generate the docs.